frontend/packages/process-editor/src/components/ConfigViewerPanel/ConfigViewerPanel.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { StudioDisplayTile, StudioSectionHeader } from '@studio/components';
+import { useBpmnContext } from '../../contexts/BpmnContext';
+import { ConfigIcon } from '../ConfigPanel/ConfigContent/ConfigIcon';
+import { getConfigTitleHelpTextKey, getConfigTitleKey } from '../../utils/configPanelUtils';
+import { useTranslation } from 'react-i18next';
+import { Alert, Heading, Paragraph } from '@digdir/design-system-react';
+import { ConfigSurface } from '../ConfigSurface/ConfigSurface';
+
+export const ConfigViewerPanel = (): React.ReactElement => {
+  return (
+    <ConfigSurface>
+      <ConfigViewerPanelContent />
+    </ConfigSurface>
+  );
+};
+
+export const ConfigViewerPanelContent = (): React.ReactElement => {
+  const { t } = useTranslation();
+  const { bpmnDetails } = useBpmnContext();
+
+  if (bpmnDetails === null || !bpmnDetails.taskType) {
+    return <ChooseElementToViewAlert />;
+  }
+
+  const configTitle = t(getConfigTitleKey(bpmnDetails.taskType));
+  const configHeaderHelpText =
+    bpmnDetails?.taskType && t(getConfigTitleHelpTextKey(bpmnDetails.taskType));
```

Comment:
Why is this check only here and not above for getting key?

Reply:
And should maybe the same method to set these texts be used as in the ConfigContent?

Reply:
Nice catch! Let me change it! 🥳